common.mk

```diff
@@ -89,3 +94,19 @@ echo\:%:
 docker-pull/%: id=$(shell docker image inspect -f "{{.Id}}" $* 2>/dev/null)
 docker-pull/%:
 	@[[ -n "${id}" ]] || { echo "retrieving $*" && docker pull $*; }
+
+${STATIC_PRESIGN_DUMMY}: ${LICENSE_FILE}
+	${MKDIR_P} $(dir $@)
+	./scripts/obtain-presigned.sh
+	touch $@
```

Comment:
Same question.

Reply:
Well I guess these are dummies. Should they actually be dummies? What happens when the presigned url expires?

Reply:
We want to re-download the presigned file if the license file changes or SUBSTRATE_VERSION changes. The presigned links are only for an hour so the presumption is that they run `make up` which generates the presigned links and immediately downloads the artifacts.

Reply:
I see now. That should work fine